### PR TITLE
Fix problematic permissions

### PR DESCRIPTION
**Pull request by umangachapagain**
- sdk 0.17.x recursively finds roles/rolebindings in bundlemanifests/
directory which causes issues in Permissions spec of CSV. So,
removing bundlemanifests/ while generating CSV and restoring it
afetrwards to bypass the recursive check.

- run `make gen-latest-csv` with new source-manifests to
get rid of the problematic permissions block.

### CODE REVIEW

File: hack/source-manifests.sh [hunk 1]
```diff
@@ -25,9 +25,12 @@ if [ -z "$NOOBAA_IMAGE" ] || [ -z "$ROOK_IMAGE" ]; then
 fi
 
 # always start fresh and remove any previous artifacts that may exist.
+rm -rf $OCS_FINAL_DIR
```
Comment (obnoxxx) [inline hack/source-manifests.sh:28]: adding `rm -rf $OCS_FINAL_DIR` here, we can also remove this from `hack/generate-unified-csv.sh`

Comment (obnoxxx) [inline hack/source-manifests.sh:28]: I don't really know if this OCS_FINAL_DIR cleanup is entierly related to the change with the bundledmanifests. It seems it should be a separate commit (in this PR).

File: hack/source-manifests.sh [hunk 2]
```diff
@@ -25,9 +25,12 @@ if [ -z "$NOOBAA_IMAGE" ] || [ -z "$ROOK_IMAGE" ]; then
...
 rm -rf $OUTDIR_TEMPLATES
 mkdir -p $OUTDIR_TEMPLATES
 mkdir -p $OUTDIR_CRDS $OUTDIR_BUNDLEMANIFESTS
+cp deploy/bundlemanifests/*.yaml $OUTDIR_BUNDLEMANIFESTS/
+rm deploy/bundlemanifests/*.yaml
```
Comment (obnoxxx) [inline hack/source-manifests.sh:33]: I think this is the main trick explained in the commit message. Since we are trying to trick the operator-sdk (if I understood it correctly), it might be good to add an explaning comment here (and at the bottom when we copy it back), since this is not obvious from reading the code.

File: hack/source-manifests.sh (hunk 1 above)
Comment (umangachapagain) [reply hack/source-manifests.sh:28]: It is related. otherwise the same issue occurs as contents of `bundlemanifests/` are copied inside `OCS_FINAL_DIR` for packaging.

**Comment by jarrpa** [comment]
We should see what we can do to avoid this workaround in the future.

File: hack/source-manifests.sh (hunk 1 above)
Comment (obnoxxx) [reply hack/source-manifests.sh:28]: Where is this happening? It is not very obvious: before this patch, OCS_FINAL_DIR was not referenced in this script at all...
